libs/category/driver/magento/src/category.service.ts

```diff
@@ -23,6 +24,7 @@ import {
 import { DaffCategoryServiceInterface } from '@daffodil/category/driver';
 import {
   DaffProductMagentoDriverConfig,
+  DAFF_PRODUCT_MAGENTO_EXTRA_PRODUCT_PREVIEW_FRAGMENTS,
```

Comment:
I don't think that this should be the same fragment token for the extension point. These two features are used differently and therefore need to be targeted separately.

Reply:
So category would have its own token and composite and configurable would depend on category?

Reply:
I misunderstood the intent here -- and I think this is a failing of docs for `@daffodil/product/driver/magento`. I'll add an issue clarifying this.